prboom2/src/doomtype.h

```diff
@@ -39,14 +39,14 @@
 #include "config.h"
 #endif
 
-#ifndef __BYTEBOOL__
-#define __BYTEBOOL__
-/* Fixed to use builtin bool type with C++. */
-#ifdef __cplusplus
-typedef bool dboolean;
-#else
-typedef enum {false, true} dboolean;
+#ifndef __DBOOLEAN__
```

Comment:
Do we need these ifndef blocks? The whole file is wrapped in an ifndef.

Reply:
Most likely not, they aren't checked anywhere else. Checking the blame, they were here from the initial revision. Would you like them removed?

Reply:
Yes please ✂️